packages/core/src/CorePlugin.ts

```diff
@@ -68,20 +62,6 @@ export class CorePlugin extends JWPlugin {
     insert(params: InsertParams): void {
         this.editor.vDocument.insert(params.value);
     }
-    /**
-     * Insert text at range.
-     *
-     * @param params
-     */
-    insertText(params: InsertTextParams): void {
```

Comment:
You forgot to move the `Params` types associated with these commands. And this PR is not rebased on master.